src/templates/Comment/schema.js

```diff
@@ -212,6 +197,51 @@ const createCommentSchema = ({
     )
   }
 
+  const blockquoteParagraph = {
+    matchMdast: matchParagraph,
+    component: BlockQuoteParagraph,
+    rules: [
+      ...globalInlines
+    ]
+  }
+
+  const getNestedRule = (levels, rule) => {
+    if (levels === 0) {
+      return rule
+    }
+    rule.rules.push(rule)
+    return getNestedRule(levels - 1, rule)
+  }
+
+  const blockQuoteNested = {
+    matchMdast: matchType('blockquote'),
+    component: BlockQuoteNested,
+    rules: [
+      heading,
+      blockquoteParagraph,
+      blockCode,
+      list,
+      thematicBreak,
+      blockLevelHtml,
+      definition
+    ]
+  }
+
+  const blockQuote = {
+    matchMdast: matchType('blockquote'),
+    component: BlockQuote,
+    rules: [
+      blockquoteParagraph,
+      heading,
+      getNestedRule(3, blockQuoteNested),
```

Comment:
there is a way to allow infinite nesting—by not providing any child rules. Just add `blockQuoteNested` here and remove the `rules` array from `blockQuoteNested`